src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,325 @@
+		assertThatThrownBy(() -> {
+			menuService.create(request);
+		}).isInstanceOf(IllegalArgumentException.class);
+
+
+	}
+
+	@Test
+	@Order(5)
+	@DisplayName("메뉴에 올라간 상품의 수량은 0개 이상이어야 합니다.")
+	void checkQuantity() {
+		//given
+		Menu request = mock(Menu.class);
+		MenuGroup menuGroup = mock(MenuGroup.class);
+		MenuProduct menuProduct = mock(MenuProduct.class);
+		Product product = mock(Product.class);
+
+		when(request.getPrice()).thenReturn(MENU_PRICE);
+		when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+		List<MenuProduct> menuProducts = Arrays.asList(menuProduct, menuProduct);
+		when(request.getMenuProducts()).thenReturn(menuProducts);
+		List<Product> productList = Arrays.asList(product, product);
+		when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+
+		//when
+		when(menuProduct.getQuantity()).thenReturn(NEGATIVE_NUM);
+
+		//then
+		assertThatThrownBy(() -> menuService.create(request))
+			.isInstanceOf(IllegalArgumentException.class);
+	}
+
+	@Test
+	@Order(6)
+	@DisplayName("메뉴 가격은 메뉴의 상품들 가격의 합보다 비쌀 수 없습니다.")
+	void comparePrice() {
+		//given
+		Menu request = mock(Menu.class);
+		MenuGroup menuGroup = mock(MenuGroup.class);
+		MenuProduct menuProduct = mock(MenuProduct.class);
+		Product product = mock(Product.class);
+
+		when(request.getPrice()).thenReturn(MENU_PRICE);
+		when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+		List<MenuProduct> menuProducts = Arrays.asList(menuProduct, menuProduct);
+		when(request.getMenuProducts()).thenReturn(menuProducts);
+		List<Product> productList = Arrays.asList(product, product);
+		when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+		when(menuProduct.getQuantity()).thenReturn(POSITIVE_NUM);
+		//when
+		when(requireNonNull(product).getPrice()).thenReturn(BigDecimal.ZERO);
+		when(productRepository.findById(any())).thenReturn(Optional.of(product));
+
+		//then
+		assertThatThrownBy(() -> menuService.create(request))
+			.isInstanceOf(IllegalArgumentException.class);
+	}
+
+	@Test
+	@Order(7)
+	@DisplayName("가게 점주는 가격을 변경할 수 있습니다.")
+	void changePriceInMenu() {
+		//given
+		Menu request = mock(Menu.class);
+		Menu menu = mock(Menu.class);
+		MenuProduct menuProduct = mock(MenuProduct.class);
+
+		when(request.getPrice()).thenReturn(MENU_PRICE);
+		when(menuRepository.findById(any())).thenReturn(Optional.ofNullable(menu));
+		List<MenuProduct> menuProductList = Arrays.asList(menuProduct, menuProduct);
+		when(requireNonNull(menu).getMenuProducts()).thenReturn(menuProductList);
+		when(menuProduct.getProduct()).thenReturn(mock(Product.class));
+		when(menuProduct.getProduct().getPrice()).thenReturn(PRODUCT_PRICE);
+		when(menuProduct.getQuantity()).thenReturn(POSITIVE_NUM);
+
+		//then
+		menuService.changePrice(RANDOM_UUID, request);
+		verify(menu).setPrice(MENU_PRICE);
+	}
+
+	@Order(8)
+	@ParameterizedTest
+	@MethodSource("menuPriceNullAndMinus")
+	@DisplayName("변경하려는 메뉴의 가격은 0원 이상입니다.")
```

Comment:
```suggestion
	@DisplayName("메뉴의 가격이 0원 미만일 경우 Exception을 던진다.")
```
테스트 코드의 의도를 명확히 나타내보세요.